### PR TITLE
SureSelect v7 support

### PR DESCRIPTION
**Pull request by melferink**
* Update from CREv2 design to SSv7 design
* Update threshold chrX for females (for SSv7)
* Update CM format field in VCF to include design
* Hardcoded path removed in defaul setting

### CODE REVIEW

File: ExomeDepth/run_ExomeDepth.py [hunk 1]
```diff
@@ -185,12 +185,12 @@ def multiprocess_call(multiprocess_list):
...
 
     action = (
-        "python {csv2vcf} {inputcsv} {refset} {model} {gender} "
+        "python {csv2vcf} {inputcsv} {refset} {calling_model} {gender} "
         " {sampleid} {template} {runid} ").format(
             csv2vcf=settings.csv2vcf,
             inputcsv=inputcsv,
             refset=args.refset,
-            model=multiprocess_list[0],
+            calling_model=multiprocess_list[7],
```
Comment (FiniDG) [inline ExomeDepth/run_ExomeDepth.py:193]: you change model to calling_model here, but you never reference it in this script, but still use "model"
https://github.com/UMCUGenetics/Dx_resources/blob/aff23bfceb0d5a606801f60e55dee21753229d08/ExomeDepth/run_ExomeDepth.py#L222
until line 250.

where is calling_model used then?

Comment (melferink) [reply ExomeDepth/run_ExomeDepth.py:193]: Yes they are. This specific change was made to include the full naming of the high-confident model (i.e. HC_SSv7-2021-1) into the VCF format field.
The other 'model' variables (HC/UMCU) are not changed  as these will impact file naming too much.

File: ExomeDepth/submit_batch_exomedepth.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def process(bam, args, refset_dic, gender_dic, suffix_dic):
     parser.add_argument('--pipeline', default='nf', choices=['nf', 'iap'], help='pipeline used for sample processing (nf = nexflow (default), IAP = illumina analysis pipeline)')
     parser.add_argument('--refset', default = settings.refset, help='Reference set to be used. Default = refset in settings.py')
     parser.add_argument('--reanalysis', help='Tab delimited file with SampleID, RefsetID, and optional reanalysis female/male mode (see settings.py) to be used. If samples are not present in the file, default refset and female/male is used')
-    parser.add_argument('--exomedepth', default = "/hpc/diaggen/software/production/Dx_resources/ExomeDepth/run_ExomeDepth.py", help='Full path to exomedepth script')
+    parser.add_argument('--exomedepth', default = "{}/run_ExomeDepth.py".format(settings.cwd), help='Full path to exomedepth script')
```
[the remark this answers is not in the document]
Comment (melferink) [reply ExomeDepth/submit_batch_exomedepth.py:70]: No it isn't.  But made some changed to make settings.py more uniform regarding "/"

**Review by FiniDG — APPROVED** [review]
After explanation of my questions in de comments and some small changes in the new commit, I approve this merge

**Pull request merged**